MANIFEST.in

```diff
@@ -1,5 +1,4 @@
 include *.txt LICENSE tox.ini .coveragerc *.rst *.toml.example OWNERS
-recursive-include gordon_janitor_gcp *.py
```

Comment:
Should the manifest also `recursive-include src *` or is that not needed since there's no non-python files?

Reply:
there's no need - manifests are for everything you want to include that are not already included in setup.py.